lib/deso_math.go

```diff
@@ -99,6 +100,7 @@ var (
 	bigOne            = NewFloat().SetUint64(1)
 	bigTwo            = NewFloat().SetUint64(2)
 	bigSqrt2Over2     = NewFloat().Quo(bigSqrt2, bigTwo)
+	bigE              = NewFloat().SetFloat64(2.71828182845904523536028747135266249775724709369995957496696763)
```

Comment:
should we use `math.E` instead? or do we like how we have precision controlled w/ the constant here?